src/pages/404.tsx

```diff
@@ -0,0 +1,31 @@
+import Link from "next/link";
+import { Flex, Button, Text } from "@chakra-ui/react";
+import UnSmile from "@/assets/svg/404.svg";
+import HOME_PAGE_LINK from "../constants";
+import NextHeadSeo from "next-head-seo";
+
+export default function Custom404() {
+  return (
+    <>
+      <NextHeadSeo
+        title={`404 Error page - Courtcanva`}
+        description={`404 Error page on CourtCanva`}
+        canonical={`http://www.courtcanva.com/404`}
+      />
+      <Flex direction="column" justify="center" align="center" h="calc(100vh - 106px)">
```

Comment:
Why 100vh - 106px? The navbar height is 72px I believe.

Reply:
Yes you are right, i have fixed it. As we considered the footer hight before.